status/templatetags/status_filters.py

```diff
@@ -121,10 +132,7 @@ def show_detail(submission, user):
     contests = get_running_contests()
```

Comment:
OK. I changed the the judgement ordering so that only contest owner/coowner can view code in contest mode.